docs/scos/dev/architecture/conceptual-overview.md

```diff
@@ -47,7 +47,7 @@ Spryker is a Commerce Operating System, mainly composed of several applications,
 
 * *Storefront*—Frontend-presentation layer for customers, provided by Yves Application Layer based on [Symfony Components](https://symfony.com/components).
 * *Back Office*—an application that contains all business logic and the backend GUI, provided by Zed Application Layer, and also uses the Symfony Components.
-* *Storefront API*—an application providing resources for customers' interaction, provided by Glue Application Layer, based on [JSON API convention](https://jsonapi.org/).
```

Comment:
TO my knowledge, JSON API is a part of a Rest API, so we have to rephrase :

application providing resources for customers' interaction, provided by Glue Application Layer, based on [REST API](https://restfulapi.net/), with a huge support in particular for [JSON API convention](https://jsonapi.org/).

Reply:
@a-sabaa what do you think?